_posts/blog/2016-12-22-dwh-part1_getting_started.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: "A Journey Towards a Custom Data Warehouse Solution <br/> Part 1 : Getting Started"
+description:
+modified:
+categories: blog
+author: tino_noack
+excerpt: We would like to share our first approach towards a custom data warehouse solution. This is based on the kinds and the amount of data we have to process. We describe DWH components and identify tools which can be applied for implementation. We also introduce a data lake as a more 'Big-Data-friendly' but provisional solution.
+tags: [Data, DWH, Data Warehouse, Big Data, Data Platform, Data Lake]
+image:
+date: 2016-12-22T00:00:00-04:00
```

Comment:
Change the date of the blog post to today, both in the name of the post, and here.